draft-ietf-rats-corim.md

```diff
@@ -2387,32 +2387,51 @@ where for each `series` entry, if the `selection` ECT matches an ACS ECT,
 the `addition` ECT is added to the ACS.
 Series iteration terminates after the first matching series entry is processed or when no series entries match.
 
-#### Processing Key Verification Endorsements {#sec-process-keys}
+#### Processing Key Verifications {#sec-process-keys}
 
-For each `ev` entry, the `condition` ECT is compared with an ACS ECT, where the ACS ECT `cmtype` contains either `evidence`, `reference-values`, or `endorsements`.
-If the ECTs match ({{sec-match-condition-ect}}), for each _key_ in `ev`.`condition`.`element-claims`.`measurement-values-map`.`intrep-keys`:
+To process key verification triples, the internal representation of ECTs containing `intrep-keys` is used to identify ACS entries containing `$crypto-key-type-choice` values that require additional key verification steps.
+If the `key-type` field is set, the Verifier will apply the verification steps defined below.
+If the key verification check succeeds, the key is re-asserted by the Verifier as an Endorsement by constructing an ECT that contains the verified key using the `authority` of the Verifier.
```

Comment:
This statement seems problematic.  In §8.1, we say that in phase 4 (i.e., this one) _"inputs are added with the Endorser's authority”_.  Verifier’s authority is added in phase 5 and is, AFAIU, out of scope of CoRIM proper.

Reply:
I think a reasonable and quick way out is to slightly modify the sentence as follows:
```suggestion
If the key verification check succeeds, the key is re-asserted by the Verifier as an Endorsement by constructing an ECT that contains the verified key using the `authority` of the Verifier.
Note that, in this case, the Verifier is acting in the role of an Endorser.
```